restli-int-test/src/test/java/com/linkedin/restli/examples/TestParseqBasedFluentClientApiWithProjections.java

```diff
@@ -117,9 +114,7 @@ public void testBatchGetWithProjection() throws Exception
 
     Set<Long> ids = Sets.newHashSet(Arrays.asList(1L, 2L, 3L));
     CompletionStage<Map<Long, EntityResponse<Greeting>>> result = greetings.batchGet(ids,
-        new GreetingsFluentClient.BatchGetOptionalParameters().setFields(
-            Greeting.createMask().withTone()
-        ));
+        optionalParams -> optionalParams.withFields(mask -> mask.withTone()));
```

Comment:
I think it would be better to either be `withMask(mask -> ...)` or `withFields(fields -> ...)`, but this is a minor consideration.

Reply:
@tjni I actually agree with you, and adding this will be sweet (sugar). This only requires adding another sets of methods, doesn't it?

Reply:
Update the API to withMask.